src/Home.js

```diff
@@ -87,119 +101,166 @@ export default class Home extends Component {
+            <Menu inline={true} responsive={false} direction='row'>
+              <Anchor href='http://slackin.grommet.io'
                 icon={<SlackIcon a11yTitle='Grommet Slack' />}/>
-              <Anchor href="https://github.com/grommet/grommet"
+              <Anchor href='https://github.com/grommet/grommet'
                 icon={<GithubIcon a11yTitle='Grommet Github' />}/>
-              <Anchor href="https://twitter.com/grommetux"
-                icon={<TwitterIcon colorIndex="grey-4"
+              <Anchor href='https://twitter.com/grommetux'
+                icon={<TwitterIcon colorIndex='grey-4'
                 a11yTitle='Grommet Twitter' />}/>
             </Menu>
           </Footer>
         </HomeSection>
 
-        <HomeSection ref="mobile"
-          backgroundImage={'url(/img/mobile_first.jpg)'} justify="start"
-          colorIndex="dark" style={mobileStyle}>
-          <Box className="home__mobile" align="center">
-            <Heading tag="h2" align="center">
+        <HomeSection ref={(ref) => this._mobileRef = ref}
+          backgroundImage={'url(/img/mobile_first.jpg)'} justify='start'
+          colorIndex='dark' style={mobileStyle}>
+          <Box className='home__mobile' align='center'>
+            <Heading tag='h2' align='center'>
               Mobile-first ready for business
             </Heading>
-            <Paragraph align="center">Think small. Starting with mobile-first
+            <Paragraph align='center'>Think small. Starting with mobile-first
               design allows for easy app scalability to larger
               devices.</Paragraph>
             {/*}
-            <NavButton path="/docs/learn" label="Learn" />
+            <NavButton path='/docs/learn' label='Learn' />
             {*/}
           </Box>
         </HomeSection>
 
+        <HomeSection colorIndex='neutral-1'>
+          <Heading tag='h2'>Why Grommet?</Heading>
+          <Box direction='row' wrap={true}
+            pad={{
+              between: 'medium', vertical: 'medium', horizontal: 'medium'
+            }}>
+            <WhyGrommetItem icon={<CubesIcon size='xlarge' />}
+              heading='Comprehensive'>
+              <Paragraph align='center' margin='none'>
+                Curated design and development platform with attention to
+                details in typography, color, spacing, and interaction.
+                Grommet is more than just a UI widget library for React.
+                It also offers templates and sample apps for higher order
+                component interaction.
+              </Paragraph>
+            </WhyGrommetItem>
+            <WhyGrommetItem icon={<ConfigureIcon size='xlarge' />}
+              heading='Flexible'>
+              <Paragraph align='center' margin='none'>
+                Responsive design without the complexity of managing
+                breakpoints and grids. Components are loaded individually
+                so that you only pay for what you use.
```

Comment:
Hi @alansouzati - when you mention the word "pay" in this sentence - is the intention of Grommet to be a "pay as you go" UI - i.e. *not* Open Source?  If not, then perhaps this is the wrong terminology?